lib/web_ui/lib/ui_web/src/ui_web/platform_view_registry.dart

```diff
@@ -4,6 +4,18 @@
 
 import 'package:ui/src/engine.dart';
 
+/// The view type of the factory that creates visible platform view DOM elements.
+///
+/// There's no need to register this view type with [PlatformViewRegistry]
+/// because it is registered by default.
+const String kDefaultVisibleViewType = '_default_document_create_element_visible';
+
+/// The view type of the factory that creates invisible platform view DOM elements.
+///
+/// There's no need to register this view type with [PlatformViewRegistry]
+/// because it is registered by default.
+const String kDefaultInvisibleViewType = '_default_document_create_element_invisible';
```

Comment:
My only change would be to make these `static final` of the `PlatformViewManager` so they're encapsulated within the class that's going to make use of them, rather than being some free globals of the library. they'd be used as `web_ui.PlatformViewManager.kDefaultInvisibleViewType` from the framework.

Reply:
Really good call!

I also found this section in our style guide that supports your comment: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k